chimney/src/test/scala/io/scalaland/chimney/DslFSpec.scala

```diff
@@ -786,5 +786,98 @@ object DslFSpec extends TestSuite {
           .transform ==> Right(OuterOut(InnerOut("pure: test")))
       }
     }
+
+    "error path" - {
```

Comment:
This file is huge enough already. WDYT about creating separate test suite for testing error path reporting for lifted transformers?

Reply:
Moved to `ErrorPathSpec` and `CatsValidatedErrorPathSpec`